.prettierrc

```diff
@@ -3,5 +3,6 @@
   "singleQuote": true,
   "jsxSingleQuote": true,
   "arrowParens": "avoid",
-  "trailingComma": "none"
+  "trailingComma": "none",
+  "endOfLine": "auto"
```

Comment:
I added this because otherwise I get an error from eslint. (It red-underlines the end of all code lines.)

Reply:
that's entirely a windows issue